packages/react-instantsearch-core/src/widgets/DynamicWidgets.tsx

```diff
@@ -23,9 +23,14 @@ function getAttribute(component: ReactChild): string | undefined {
 type DynamicWidgetsProps = {
   children: ReactNode;
   attributesToRender: string[];
+  fallbackWidget?: ComponentType<{ attribute: string }>;
 };
 
-function DynamicWidgets({ children, attributesToRender }: DynamicWidgetsProps) {
+function DynamicWidgets({
+  children,
+  attributesToRender,
+  fallbackWidget: Fallback = () => null,
```

Comment:
`widget` doesn't sound like React. Maybe just `fallback` or `fallbackComponent` (as in `hitComponent` in `Hits`)?

Reply:
good point! I put this in the description as well, since I'm not sure what's best. At the moment I'm edging towards fallbackComponent